server/lib/ldap/Connection.ts

```diff
@@ -545,12 +545,18 @@ export class LDAPConnection {
 		}
 
 		bindLogger.info({ msg: 'Binding UserDN', userDN: this.options.authenticationUserDN });
-		await this.bindDN(this.options.authenticationUserDN, this.options.authenticationPassword);
-		this.usingAuthentication = true;
+		try {
+			await this.bindDN(this.options.authenticationUserDN, this.options.authenticationPassword);
+			this.usingAuthentication = true;
+		} catch (error) {
+			authLogger.info({ msg: 'Base Authentication Issue', dn: this.options.authenticationUserDN });
+			authLogger.debug({ msg: 'error', error });
```

Comment:
any reason this is not an "error" log? like should this be visible only if log level is set to INFO or DEBUG? if so, just discard my suggestion =)
```suggestion
			authLogger.error({ msg: 'Base Authentication Issue', err: error, dn: this.options.authenticationUserDN });
```

Reply:
@sampaiodiego no idea, I just followed other examples in the code. @pierre-lehnen-rc WDYT?